src/router/routes.ts

```diff
@@ -22,59 +23,27 @@ export const routes = [
     }
   },
   {
-    path: '/define-company',
-    name: RouteNames.DEFINE_COMPANY,
-    component: DefineCompany,
+    path: '/correction',
+    name: RouteNames.CORRECTION,
+    component: Correction,
     meta: {
-      step: 1,
-      label: 'Define Your Company',
-      requiresAuth: true
+      requiresAuth: true,
+      isStaffOnly: true
     }
   },
   {
-    path: '/add-people-roles',
-    name: RouteNames.ADD_PEOPLE_AND_ROLES,
-    component: AddPeopleAndRoles,
+    path: '/alteration',
+    name: RouteNames.ALTERATION,
+    component: Alteration,
     meta: {
-      step: 2,
-      label: 'Add People and Roles',
-      requiresAuth: true
-    }
-  },
-  {
-    path: '/create-share-structure',
-    name: RouteNames.CREATE_SHARE_STRUCTURE,
-    component: CreateShareStructure,
-    meta: {
-      step: 3,
-      label: 'Create Share Structure',
-      requiresAuth: true
-    }
-  },
-  {
-    path: '/incorporation-agreement',
-    name: RouteNames.INCORPORATION_AGREEMENT,
-    component: IncorporationAgreement,
-    meta: {
-      step: 4,
-      label: 'Incorporation Agreement',
-      requiresAuth: true
-    }
-  },
-  {
-    path: '/review-confirm',
-    name: RouteNames.REVIEW_CONFIRM,
-    component: ReviewConfirm,
-    meta: {
-      step: 5,
-      label: 'Review and Confirm',
-      requiresAuth: true
+      requiresAuth: true,
+      isStaffOnly: false
     }
   },
   {
     // default/fallback route
     // must be last
     path: '*',
-    redirect: '/define-company'
+    redirect: '/correction'
```

Comment:
Not sure what the fallback route should be, so Corrections for now.

Reply:
Sounds reasonable.
What happens if there is no default/fallback route and you go an undefined route?

Reply:
Good question, i'll find out!

Reply:
I'm guessing you'll get a 404... this is OK with me.
I'm thinking that defaulting to a "profile" (read-only) page would be a safe fallback as well.
Maybe this can be sorted out in the (near) future, not now.

Reply:
In this case, we mount the app but get stuck on our infinite loading spinner because we can't reach our views. 
The refactor puts the data loading for the respective filings into our views which in turn let the app know its ready or not ready. So without that, infinite loader

Reply:
Makes sense.
I can't decide what's best (or worst) for now, so do whatever and we can rethink this when we know more :)